ui/src/containers/Header/index.tsx

```diff
@@ -28,7 +28,7 @@ const Header: React.FC = observer(() => {
           <Search />
         </GridItem>
       </Grid>
-      {user.isAuthenticated ? (
+      {user.isAuthenticated && user.refreshTokenInfo.expiresAt * 1000 > global.Date.now() ? (
```

Comment:
What wil lhappen if `user.refreshTokenInfo.expiresAt * 1000 `< global.Date.now()`

Reply:
user will be logged out

Reply:
So new refresh token api won't be called ??

Reply:
yes